samples/v1/types_config.go

```diff
@@ -66,36 +66,50 @@ type ConfigStatus struct {
 	// managementState reflects the current operational status of the on/off switch for
 	// the operator.  This operator compares the ManagementState as part of determining that we are turning
 	// the operator back on (i.e. "Managed") when it was previously "Unmanaged".
+	// +patchMergeKey=type
+	// +patchStrategy=merge
```

Comment:
@adambkaplan @dmage also this might need `// +optional` ?

@openshift/api-reviewers who reviewed this api? ;-)